cmd/mqtt2prometheus.go

```diff
@@ -64,6 +64,11 @@ var (
 		"",
 		"[EXPERIMENTAL] Path to configuration file that can enable TLS or authentication for metric scraping.",
 	)
+	usePasswordFromFile = flag.Bool(
+		"use-secret",
```

Comment:
Can we rename this flag? Sure, the docker secret is one usecase, another could be some service like a hashicorp vault agent, creating this file. I would like to rename it to:
```suggestion
		"treat-mqtt-password-as-file-name",
```
This clearly indicates, what this flag does. Yes, it's long, but not typed by humans frequently.

Reply:
agreed, it makes more sense to call it out clearly.